### DISCUSSION TITLE
Ram and swap fill up

### DISCUSSION THREAD

**Discussion by goffy76** [comfyanonymous/ComfyUI#5251] [Q&A]
Hi, I'm completely new to Linux and SD/ComfyUI and I have a problem. I have got ComfyUI running on Ubuntu 22.04 (can't get it to work on 24.04) My problem is that when I change model/checkpoint my RAM and my swap file fill up and then the machine freezes. I have increased my swap to 16GB but it doesn't use more than about 6.5GB and it still freezes. It is especially bad when I try to load a Flux model. In fact, it is never successful.

Info: I have 16 GB RAM and an AMD RX7800XT with 16 GB VRAM and it runs fine - once I have loaded a model.

What does a newbie do??

**Comment by ltdrdata** [comment]
16GB is too small for FLUX.
